public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -3,9 +3,9 @@ import _ from 'lodash';
 import $ from 'jquery';
 // Services & Utils
 import kbn from 'app/core/utils/kbn';
-import { dateMath, TimeRange, DateTime, AnnotationEvent } from '@grafana/data';
-import { Observable, from, of, merge } from 'rxjs';
-import { filter, map } from 'rxjs/operators';
+import { AnnotationEvent, dateMath, DateTime, LoadingState, TimeRange } from '@grafana/data';
```

Comment:
Honestly no idea why this gets ordered all the time. I assume Goland does something when autoimporting stuff need to figure that out.